job_spec/WATER_MAP_TEST.yml

```
@@ -0,0 +1,191 @@
+            - description: The name of the IW VV+VH Sentinel-1 SLC granule to process
+              type: string
+              pattern: "^S1[AB]_IW_SLC__1SDV"
+              minLength: 67
+              maxLength: 67
+              example: S1A_IW_SLC__1SDV_20211110T234815_20211110T234842_040516_04CE0A_E717
+    bucket_prefix:
+      default:  '""'
+    resolution:
+      api_schema:
+        default:  30.0
+        description: Desired output pixel spacing in meters
+        type: number
+        enum:
+          - 30.0
+          - 20.0
+          - 10.0
+    speckle_filter:
+      api_schema:
+        description: Apply an Enhanced Lee speckle filter
+        default: false
+        type: boolean
+    max_vv_threshold:
+      api_schema:
+        description: Maximum threshold value to use for VV polarized raster in decibels (dB)
+        default: -15.5
+        type: number
+    max_vh_threshold:
+      api_schema:
+        description: Maximum threshold value to use for VH polarized raster in decibels (dB)
+        default: -23.0
+        type: number
+    hand_threshold:
+      api_schema:
+        description: The maximum height above nearest drainage in meters to consider a pixel valid
+        default: 15.0
+        type: number
+    hand_fraction:
+      api_schema:
+        description: The minimum fraction of valid HAND pixels required in a tile for thresholding
+        default: 0.8
+        type: number
+    membership_threshold:
+      api_schema:
+        description: The average membership to the fuzzy indicators required for a water pixel
+        default: 0.45
+        type: number
+    flood_depth_estimator:
+      api_schema:
+        description: Flood depth estimation approach. A value of None indicates that flood depth estimation will not be performed.
+        default: None
+        type: string
+        enum:
+          - iterative
+          - logstat
+          - nmad
+          - numpy
+          - None
+    water_level_sigma:
+      api_schema:
+        description: Standard deviation to estimate max water height for each object. Ignored when flood_depth_estimator is None.
+        default: 3.0
+        type: number
+    known_water_threshold:
+      api_schema:
+        description: Threshold for extracting known water area in percent. Ignored when flood_depth_estimator is None.
+        default: None
+        type: number
+    iterative_min:
+      api_schema:
+        description: Minimum bound used for iterative method. Ignored when flood_depth_estimator is None.
+        default: 0
+        type: integer
+    iterative_max:
+      api_schema:
+        description: Maximum bound used for iterative method. Ignored when flood_depth_estimator is None.
+        default: 15
+        type: integer
+    minimization_metric:
+      api_schema:
+        description: Evaluation method to minimize. A value of None indicates that a threshold level will be calculated.
```

```suggestion
        description: Evaluation method to minimize.
```
Oh looks like this was meant for the known water threshold paramter

---

Also would be good to descirbe this a bit better, including what fmi and ts are